modules/services/sleepwatcher/default.nix

```diff
@@ -0,0 +1,155 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.sleepwatcher;
+in
+
+{
+  options = {
+    services.sleepwatcher = {
+      enable = mkOption {
```

Comment:
You should use `mkEnableOption` here, it basically does the same thing that you do manually.
ref: https://noogle.dev/f/lib/mkEnableOption